src/playground/views/main.handlebars

```diff
@@ -149,6 +149,7 @@
 							<script>
 								const editor = registerAskScriptEditor("editor", "run", "{{askscriptServerUrl}}");
 								generateDemoMenuEntries(editor);
+								confirmExitFromPlayground(editor);
```

Comment:
Can you make it so this will work only if the editor is dirty?

Reply:
I can add this event on change in editor like editor.on('change', () => onbeforeunload ... )

Reply:
Sounds good.
Just need to make sure there's only one beforeunload event and not one per change event.
You could use your current code and maybe just during onchange add a dirty flag to the editor. Then check it in the beforeunload event?

Reply:
I like the @YonatanKra's suggestion

Reply:
@YonatanKra @mhagmajer I think we can resolve this CR. Fill free to write another :P